apps/qsim_qtrajectory_cuda.cu

```diff
@@ -251,6 +251,7 @@ int main(int argc, char* argv[]) {
   typename QTSimulator::Parameter param3;
   param3.max_fused_size = opt.max_fused_size;
   param3.verbosity = opt.verbosity;
+  param3.apply_last_deferred_ops = true;
```

Comment:
Should this be false? The other changes in this file seem built around the optimization in this PR.

Reply:
This should be true (this is the default value; just for self documenting). It is set to false below after the first primary trajectory.